service/collector/vpc.go

```diff
@@ -87,10 +97,17 @@ func (c *Collector) collectVPCs(ch chan<- prometheus.Metric) {
 			}
 		}
 
+		accountID, err := c.awsAccountID(awsClients)
+		if err != nil {
+			c.logger.Log("level", "error", "message", "could not get aws account id", "stack", fmt.Sprintf("%#v", err))
```

Comment:
There is not much we can do about those errors right now.
One improvement consist on having a "success" metrics triggering after all metrics got successfully collected. Like [prometheus/node_exporter](https://github.com/prometheus/node_exporter/blob/24a28fcc9e9123b3b39058918f98b64a1938a0a9/collector/collector.go#L37) is doing.